odc-product-delete/delete_odc_product_explorer.sql

```diff
@@ -28,6 +38,57 @@ SELECT count(*) FROM cubedash.product WHERE name = :'product_name';
 
 DELETE FROM cubedash.product WHERE name = :'product_name';
 
+--
+-- delete lineage bond product
```

Comment:
What's this section intending to do? It looks like its deleting all products that are upstream or downstream of the given product... are we sure that's what we want? 

I think it'll probably fail? (its not deleting their regions etc. either)

Reply:
Yeah, I don't think we should be considering automatically deleting lineage products, if that's what "lineage bond" means.

Deleting a product that has a dependency should fail, fast. And deleting a dependency of a deleted product can be a second step. Automating this stuff if fraught with danger.

Reply:
Agreed.

We probably only need an update statement to remove the deleted product from any `derived_products_ref` arrays (ie. it's own source products). If it has any derived products itself it should fail.

Reply:
thanks everyone, i thought about and i didn't think too much about it, its been changed to remove that linkage from `source_product_refs` and `derived_product_refs`